RecoTracker/TkSeedingLayers/src/SeedingLayerSetsBuilder.cc

```diff
@@ -48,7 +48,7 @@ SeedingLayerSetsBuilder::SeedingLayerId SeedingLayerSetsBuilder::nameToEnumId(co
     //
     else if ((index = name.find("FPix")) != string::npos) {
       subdet = GeomDetEnumerators::PixelEndcap;
-      idLayer = atoi(name.substr(index+4,1).c_str());
+      idLayer = atoi(name.substr(index+4).c_str());
```

Comment:
The atoi() function converts the initial portion of the string pointed to by str to int representation.
tested with only 'FPix8+9+10+11' combination in the forward.